lib/entities/role.ts

```diff
@@ -12,13 +12,15 @@ export type RoleProps = {
    * Permissions for application sections
    */
   permissions: {
-    ContentDelivery: string
+    ContentDelivery: string[] | string
     ContentModel: string[]
-    Settings: any[]
+    EnvironmentAliases: string[] | string
+    Environment: string
+    Settings: any[] | string
   }
   policies: {
     effect: string
-    actions: string
+    actions: string[] | string
```

Comment:
Looks like actions list is a limited set of actions:

```
"read",
"create",
"update",
"delete",
"publish",
"unpublish",
"archive",
"unarchive"
```

so it can be 'all' | ActionType[]